examples/stocks/lib/stock_strings.dart

```diff
@@ -8,20 +8,26 @@ part of stocks;
 // To generate the stock_messages_*.dart files from the ARB files, run:
 //   pub run intl:generate_from_arb --output-dir=lib/i18n --generated-file-prefix=stock_ --no-use-deferred-loading lib/stock_strings.dart lib/i18n/stocks_*.arb
 
-class StockStrings {
-  static String title() => Intl.message(
+class StockStrings extends LocaleQueryData {
+  static StockStrings of(BuildContext context) {
+    return LocaleQuery.of(context) as StockStrings;
+  }
+
+  static final StockStrings instance = new StockStrings();
```

Comment:
This is mostly for efficiency - StockStrings is stateless so it seemed unnecessary to potentially allocate multiple instances.  It's also convenient to have a singleton here in case you need access to the strings outside of a BuildContext.